docs/http-api/swagger.yaml

```diff
@@ -0,0 +1,2495 @@
+        - name: syncOnly
+          in: query
+          description: When true, prevents a new scavenge point from being written.
+          type: boolean
+          required: false
+          default: false
+      responses:
+        "200":
+          description: OK
+        "400":
+          description: Bad request
+        "401":
+          description: Unauthorized
+      security:
+        - basicAuth: []
+  "/admin/scavenge/{scavengeId}":
+    delete:
+      description: Stop a running scavenge operation.
+      summary: Stop a scavenge operation
+      tags:
+        - Admin
+      operationId: Stop a scavenge
+      produces:
+        - application/json
+      parameters:
+        - name: scavengeId
+          in: path
+          description: The scavenge ID
+          required: true
+          type: integer
+      responses:
+        "200":
+          description: OK
+        "401":
+          description: Unauthorized
+      security:
+        - basicAuth: []
+  /admin/scavenge/current:
+    get:
+      description: Returns a link to the current scavenge.
+      summary: Get current scavenge
+      tags:
+        - Scavenge
+      operationId: Get current scavenge
+      parameters: []
+      produces:
+        - application/json
+      responses:
+        "200":
+          description: OK
+          schema:
+            $ref: "#/definitions/ScavengeCurrentItem"
+        "401":
+          description: Unauthorized
+      security:
+        - basicAuth: []
+  /admin/mergeindexes:
+    delete:
+      description: Manually merge indexes after a scavenge operation
+      summary: Merge indexes
+      tags:
+        - Admin
+      operationId: Merge Indexes
+      produces:
+        - application/json
+      responses:
+        "200":
+          description: OK
+        "401":
+          description: Unauthorized
+      security:
+        - basicAuth: []
+  /info:
+    get:
+      description: Returns information about node.
+      summary: Get info for node
+      tags:
+        - Info
+      operationId: Get info for node
+      produces:
+        - application/json
+      parameters: []
+      responses:
+        "200":
+          description: OK
+        "401":
+          description: Unauthorized
+      security:
+        - basicAuth: []
+  /info/options:
+    get:
+      description: Returns configuration details about node.
+      summary: Get configuration for node
+      tags:
+        - Info
+      operationId: Get configuration for node
+      produces:
+        - application/json
+      parameters: []
+      responses:
+        "200":
+          description: OK
+        "401":
+          description: Unauthorized
+      security:
+        - basicAuth: []
+  /metrics:
+    get:
+      description: Returns the metrics of a node in OpenMetrics format.
+      summary: Node metrics
+      tags:
+        - Stats
+      operationId: Node metrics
+      parameters: []
+      produces:
+        - application/openmetrics-text
```

Comment:
should this be the actual content-type header of the response? if so it looks like it is text/plain

Reply:
hmm we specify it in the controller like that, but see it's not used indeed.